meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/resin-proxy-config.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Resin proxy configuration service
+Requires=mnt-boot.mount etc-redsocks.mount
+After=mnt-boot.mount etc-redsocks.mount
+
+[Service]
+ExecStart=@BASE_BINDIR@/bash @BINDIR@/resin-proxy-config
+Type=simple
+RemainAfterExit=yes
+StandardError=journal+console
```

Comment:
No idea really :/
Again, copied from `resin-net-config` - it probably allows errors to be shown on the console (display) if something fails? Sounds useful for debugging connectivity problems behind a proxy.
@resin-os/resin-devices-team can confirm if it's useful or not.